internal/engine/inputloader.go

```diff
@@ -367,25 +387,21 @@ func (il *InputLoader) logger() InputLoaderLogger {
 	return log.Log
 }
 
-// stringListToModelURLInfo is an utility function to convert
+// inputLoaderStringListToModelExperimentTarget is an utility function to convert
 // a list of strings containing URLs into a list of model.URLInfo
 // which would have been returned by an hypothetical backend
 // API serving input for a test for which we don't have an API
 // yet (e.g., stunreachability and dnscheck).
-func stringListToModelURLInfo(input []string, err error) ([]model.OOAPIURLInfo, error) {
+func inputLoaderStringListToModelExperimentTarget(input []string, err error) ([]model.ExperimentTarget, error) {
 	if err != nil {
 		return nil, err
 	}
-	var output []model.OOAPIURLInfo
+	var output []model.ExperimentTarget
 	for _, URL := range input {
 		if _, err := url.Parse(URL); err != nil {
 			return nil, err
 		}
-		output = append(output, model.OOAPIURLInfo{
-			CategoryCode: "MISC", // hard to find a category
-			CountryCode:  "XX",   // representing no country
```

Comment:
Yeah, it's what check-in would do: https://github.com/ooni/backend/blob/master/api/ooniapi/prio.py.

So, I guess we should revert back to using "ZZ", since it's what we'd get if check-in didn't know the country.

Reply:
Fixed by https://github.com/ooni/probe-cli/pull/1615/commits/58304367bc14b7b8bada1ff7848492a75bdf4a8c